src/Controls/src/SourceGen/compat.cs

```diff
@@ -0,0 +1,43 @@
+//allow usage of required and record in netstandard2.0
+
+using System.ComponentModel;
+
+namespace System.Runtime.CompilerServices
```

Comment:
If these were copied from .NET, sometimes I just keep a copy of their header like:
https://source.dot.net/#Microsoft.CodeAnalysis.Workspaces/src/Dependencies/Contracts/RequiredMemberAttribute.cs,1